server/lib/express.js

```diff
@@ -77,9 +90,8 @@ export default async function (app) {
   // Setup session (required by passport)
 
   const redisClient = await createRedisClient();
-  if (redisClient) {
-    const store = new RedisStore({ client: redisClient });
-
+  if (redisClient || process.env.OC_ENV === 'development') {
+    const store = !redisClient ? undefined : new RedisStore({ client: redisClient });
```

Comment:
Had to change this because `@superfaceai/passport-twitter-oauth2` relies on session